Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Homework 1 and 2 
+            //1
+            string name, surename;
```

Comment:
Do not declare all local variables in one place. It's a bad practice, because this makes you switching from one context to another (from assignment back to declaration). This makes it harder to remember what the types were, forces us to keep track of a bigger code context.

Declare the variables as close as where you use them.